api/TouchList.json

```diff
@@ -54,54 +54,6 @@
           "deprecated": false
         }
       },
-      "identifiedTouch": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TouchList/identifiedTouch",
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/TouchList/identifiedTouch is 404 👍